cantools/db/database.py

```diff
@@ -34,10 +34,10 @@ def __init__(self,
         self._version = version
         self._attribute_definitions = (attribute_definitions
                                        if attribute_definitions
-                                       else [])
+                                       else {})
         self._attribute_definition_defaults = (attribute_definition_defaults
                                                if attribute_definition_defaults
-                                               else [])
+                                               else {})
```

Comment:
Make the same change in https://github.com/eerimoq/cantools/blob/master/cantools/db/internal_database.py.

Reply:
Addressed in 34d11f81e538a5b8399cbb58477c67716b693977